### ISSUE TITLE
Add the "%" Chord Symbol to repeat the previous chord

### ISSUE THREAD

**Issue by lgvr123** [musescore/MuseScore#21743]
### Your idea

Allow the "%" as chord symbol formatting option and play the previous chord when encountered during playback.

### Problem to be solved

In jazz music sheets, the "%" is often used for clarity to say "repeat previous chord".  

![rpeatChordSymbol](https://github.com/musescore/MuseScore/assets/50494/306b6546-692e-4723-b2f5-5cea71ff16e3)

This sign is at the same time convenient for the musician because the repetition is made really obvious and for writer, because he has not to repeat the previous chord and can experiment without having to change the chord at several places.

Today, the sign "%" can be entered as a Chord Symbol, but MuseScore doesn't play as it considers that sign as an error. And therefore pieces where chords are repeated are not played back correctly. The workaround is to retype all the chord at every repetition but then we loose all the benefits of this sign.

### Prior art

iRealPro supports that sign:

![Freddie Freeloader](https://github.com/musescore/MuseScore/assets/50494/a13192a6-8e64-4c6c-a802-e3061a3db9ab)

**Comment by ralphm** [comment]
This repeat-last-measure sign already exists in the “Repeats & jumps” palette:

![image](https://github.com/musescore/MuseScore/assets/116741/009754d5-72a2-4280-9b5d-e1f23ad6a95b)

Unfortunately, though, it seems that this causes all kinds of havoc with playback of the chords in my 4.2.1 install on Linux. The chords, attached to the rests in the second staff, are not repeated when they should be, interfering with the next chord or playing even long after.

For debugging purposes, this is the file I created: https://musescore.com/user/8377306/scores/14769490/s/scrKtP.

**Comment by lgvr123** [comment]
> This repeat-last-measure sign already exists in the “Repeats & jumps” palette:

Not exactly. It exists at the **measure level**. Not at the **chord symbol-level**.

If you want to reproduce this version of "Witch Hunt" (Wayne Shorter):
![Witch Hunt RealBook](https://github.com/musescore/MuseScore/assets/50494/15c7548b-3298-4ca6-a7a1-6eb7f577dc42)

using "%" at the measure-level will not work. You need to be able to use it at the chord symbol-level:
![Witch Hunt](https://github.com/musescore/MuseScore/assets/50494/f0b1cfc1-1bb5-4e2e-b1b4-17352daf2cd1)

Today, using that symbol at the chord symbol-level is feasible (although marked as error during error). 

But MU **doesn't play any sound** when encountering that symbol (as it is considering it as an error).  This is hat about is this feature request.

**Comment by blisst** [comment]
I also want this feature, see it requested numerous times in the Musescore forums:
https://musescore.org/en/node/333521
https://musescore.org/en/node/272509
https://musescore.org/en/node/84066

**Comment by mduboule** [comment]
Would love to see this implemented!

**Comment by sergeremy** [comment]
I too would very much like this feature to be implemented.

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/378393

**Comment by danmarei** [comment]
Also here looking for this feature, thanks!

**Comment by cbjeukendrup** [comment]
If you also want a certain feature, please add a 👍 to the issue (i.e. topmost comment on the page), instead of commenting on it. Only comment on something if you have useful information to add.
